01:01:27 PM: INFO [logger_base.py:13] Mensaje a nivel info
01:01:27 PM: WARNING [logger_base.py:14] Mensaje a nivel warning
01:01:27 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
01:01:27 PM: CRITICAL [logger_base.py:16] Mensaje a nivel crítico
04:56:32 PM: DEBUG [logger_base.py:12] Mensaje a nivel debug
04:56:32 PM: INFO [logger_base.py:13] Mensaje a nivel info
04:56:32 PM: WARNING [logger_base.py:14] Mensaje a nivel warning
04:56:32 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
04:56:32 PM: CRITICAL [logger_base.py:16] Mensaje a nivel crítico
04:56:36 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
04:56:36 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001C6BBB7F4F0>
04:56:36 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001C6BA040E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
04:56:36 PM: DEBUG [usuario_dao.py:14] Seleccionando usuarios...
04:56:36 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
04:56:36 PM: DEBUG [cursor_pool.py:22] Commit de transacción
04:56:36 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001C6BA040E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
04:56:36 PM: DEBUG [usuario_dao.py:41] Usuario: 1, jperez, 123
04:56:36 PM: DEBUG [usuario_dao.py:41] Usuario: 2, kgomez, 456
04:56:44 PM: INFO [logger_base.py:13] Mensaje a nivel info
04:56:44 PM: WARNING [logger_base.py:14] Mensaje a nivel warning
04:56:44 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
04:56:44 PM: CRITICAL [logger_base.py:16] Mensaje a nivel crítico
04:57:15 PM: INFO [usuario_dao.py:41] Usuario: 1, jperez, 123
04:57:15 PM: INFO [usuario_dao.py:41] Usuario: 2, kgomez, 456
05:01:51 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:02:30 PM: INFO [servicio_dao.py:41] Servicio: 1, CFE, Pago 21 de febrero, None
05:02:30 PM: INFO [servicio_dao.py:41] Servicio: 2, Agua para alberca, Pago 15 de febrero, Checar que llegue el pedido completo
05:03:32 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:03:40 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:04:52 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:05:06 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:06:55 PM: INFO [servicio_dao.py:41] Servicio: 1, CFE, Pago 21 de febrero, None
05:06:55 PM: INFO [servicio_dao.py:41] Servicio: 2, Agua para alberca, Pago 15 de febrero, Checar que llegue el pedido completo
05:07:24 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:08:11 PM: INFO [servicio_dao.py:41] Servicio: 1, CFE, Pago 21 de febrero, None
05:08:11 PM: INFO [servicio_dao.py:41] Servicio: 2, Agua para alberca, Pago 15 de febrero, Checar que llegue el pedido completo
05:11:04 PM: INFO [servicio_dao.py:42] Servicio: 1, CFE, Pago 21 de febrero, None
05:11:04 PM: INFO [servicio_dao.py:42] Servicio: 2, Agua para alberca, Pago 15 de febrero, Checar que llegue el pedido completo
05:15:40 PM: INFO [servicio_dao.py:50] Servicio: 1, CFE, Pago 21 de febrero, None
05:15:40 PM: INFO [servicio_dao.py:50] Servicio: 2, Agua para alberca, Pago 15 de febrero, Checar que llegue el pedido completo
09:22:23 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:23 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:22:27 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:27 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:22:28 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:28 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:22:29 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:29 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:22:30 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:30 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:22:30 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:30 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:22:30 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:30 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:22:30 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:30 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:22:31 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:31 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:22:31 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:31 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:22:57 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:22:57 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:37:15 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:37:15 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:39:25 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:39:25 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
09:39:26 AM: INFO [menu.py:12] Usuario: 1, jperez, 123
09:39:26 AM: INFO [menu.py:12] Usuario: 2, kgomez, 456
01:01:23 PM: INFO [menu.py:12] Usuario: 1, jperez, 123
01:01:23 PM: INFO [menu.py:12] Usuario: 2, kgomez, 456
01:01:25 PM: INFO [menu.py:12] Usuario: 1, jperez, 123
01:01:25 PM: INFO [menu.py:12] Usuario: 2, kgomez, 456
04:41:36 PM: INFO [usuario_dao.py:68] Usuario: 1, jperez, 123
04:43:07 PM: INFO [usuario_dao.py:67] Usuario: 1, jperez, 123
04:43:22 PM: INFO [usuario_dao.py:67] Usuario: jperez, 123, None
04:43:46 PM: INFO [usuario_dao.py:67] Usuario: 1, jperez, 123
04:50:55 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
04:52:53 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
04:53:06 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
04:53:59 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
04:54:05 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
04:54:10 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
04:54:34 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
04:54:47 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
04:56:40 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
04:57:21 PM: INFO [usuario_dao.py:70] Usuario: 1, jperez, 123
05:07:11 PM: INFO [usuario_dao.py:64] 1
05:09:02 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:10:10 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:10:27 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:11:12 PM: INFO [usuario_dao.py:70] Usuario: 4, rmora, 13579
05:11:12 PM: INFO [usuario_dao.py:70] Usuario: 5, rmora, 13579
05:11:37 PM: DEBUG [logger_base.py:12] Mensaje a nivel debug
05:11:37 PM: INFO [logger_base.py:13] Mensaje a nivel info
05:11:37 PM: WARNING [logger_base.py:14] Mensaje a nivel warning
05:11:37 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
05:11:37 PM: CRITICAL [logger_base.py:16] Mensaje a nivel crítico
05:11:43 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:11:44 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x00000199C60F43D0>
05:11:44 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x00000199C4793E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:11:44 PM: DEBUG [usuario_dao.py:15] Seleccionando usuarios...
05:11:44 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:11:44 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:11:44 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x00000199C4793E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:11:44 PM: DEBUG [usuario_dao.py:64] Usuario: 1, jperez, 123
05:11:44 PM: DEBUG [usuario_dao.py:64] Usuario: 2, kgomez, 456
05:11:44 PM: DEBUG [usuario_dao.py:64] Usuario: 4, rmora, 13579
05:11:44 PM: DEBUG [usuario_dao.py:64] Usuario: 5, rmora, 13579
05:11:44 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:11:44 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x00000199C4793E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:11:44 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmora, None
05:11:44 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:11:44 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:11:44 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x00000199C4793E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:11:44 PM: INFO [usuario_dao.py:70] Usuario: 4, rmora, 13579
05:11:44 PM: INFO [usuario_dao.py:70] Usuario: 5, rmora, 13579
05:14:43 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:14:43 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001E3BAE1B490>
05:14:43 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001E3B92D7E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:14:43 PM: DEBUG [usuario_dao.py:15] Seleccionando usuarios...
05:14:43 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:14:43 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:14:43 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001E3B92D7E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:14:43 PM: DEBUG [usuario_dao.py:64] Usuario: 1, jperez, 123
05:14:43 PM: DEBUG [usuario_dao.py:64] Usuario: 2, kgomez, 456
05:14:43 PM: DEBUG [usuario_dao.py:64] Usuario: 4, rmora, 13579
05:14:43 PM: DEBUG [usuario_dao.py:64] Usuario: 5, rmora, 13579
05:14:43 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:14:43 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001E3B92D7E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:14:43 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, mora, None
05:14:43 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:14:43 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:14:43 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001E3B92D7E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:16:34 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:16:34 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001BF1DCD2850>
05:16:34 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001BF1C196E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:16:34 PM: DEBUG [servicio_dao.py:14] Seleccionando servicios...
05:16:34 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:16:34 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:16:34 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001BF1C196E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:16:34 PM: DEBUG [servicio_dao.py:50] Servicio: 1, CFE, Pago 21 de febrero, None
05:16:34 PM: DEBUG [servicio_dao.py:50] Servicio: 2, Agua para alberca, Pago 15 de febrero, Checar que llegue el pedido completo
05:16:47 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:16:48 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000014351E2B490>
05:16:48 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x00000143502F9E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:16:48 PM: DEBUG [usuario_dao.py:15] Seleccionando usuarios...
05:16:48 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:16:48 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:16:48 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x00000143502F9E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:16:48 PM: DEBUG [usuario_dao.py:64] Usuario: 1, jperez, 123
05:16:48 PM: DEBUG [usuario_dao.py:64] Usuario: 2, kgomez, 456
05:16:48 PM: DEBUG [usuario_dao.py:64] Usuario: 4, rmora, 13579
05:16:48 PM: DEBUG [usuario_dao.py:64] Usuario: 5, rmora, 13579
05:16:48 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:16:48 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x00000143502F9E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:16:48 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, mora, None
05:16:48 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:16:48 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:16:48 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x00000143502F9E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:17:10 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:17:10 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x00000233662BB490>
05:17:10 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x0000023364789E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:17:10 PM: DEBUG [usuario_dao.py:15] Seleccionando usuarios...
05:17:10 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:17:10 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:17:10 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x0000023364789E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:17:10 PM: DEBUG [usuario_dao.py:64] Usuario: 1, jperez, 123
05:17:10 PM: DEBUG [usuario_dao.py:64] Usuario: 2, kgomez, 456
05:17:10 PM: DEBUG [usuario_dao.py:64] Usuario: 4, rmora, 13579
05:17:10 PM: DEBUG [usuario_dao.py:64] Usuario: 5, rmora, 13579
05:17:10 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:17:10 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x0000023364789E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:17:10 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmora, None
05:17:10 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:17:10 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:17:10 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x0000023364789E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:17:10 PM: INFO [usuario_dao.py:70] Usuario: 4, rmora, 13579
05:17:10 PM: INFO [usuario_dao.py:70] Usuario: 5, rmora, 13579
05:17:24 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:17:25 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001C4BB4B54F0>
05:17:25 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001C4BB42B9D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:17:25 PM: DEBUG [servicio_dao.py:14] Seleccionando servicios...
05:17:25 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:17:25 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:17:25 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001C4BB42B9D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:17:30 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:17:30 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001C4BB42B9D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:17:30 PM: DEBUG [usuario_dao.py:51] Buscando: rmora
05:17:30 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:17:30 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:17:30 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001C4BB42B9D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:18:19 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:18:19 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001C4BB42B9D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:18:19 PM: DEBUG [usuario_dao.py:51] Buscando: rmora
05:18:19 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:18:19 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:18:19 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001C4BB42B9D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:20:56 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:20:56 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001C4BB42B9D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:20:56 PM: DEBUG [usuario_dao.py:51] Buscando: rmora
05:20:56 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:20:56 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:20:56 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001C4BB42B9D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:21:45 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:21:45 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000018497D24100>
05:21:45 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x0000018497C73D00; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:21:45 PM: DEBUG [servicio_dao.py:14] Seleccionando servicios...
05:21:45 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:21:45 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:21:45 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x0000018497C73D00; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:21:56 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:21:56 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x0000018497C73D00; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:21:56 PM: DEBUG [usuario_dao.py:51] Buscando: rmora
05:21:56 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:21:56 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:21:56 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x0000018497C73D00; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:22:03 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:22:03 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x0000018497C73D00; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:22:03 PM: DEBUG [usuario_dao.py:51] Buscando: rmora
05:22:03 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:22:03 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:22:03 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x0000018497C73D00; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:22:04 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:22:04 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x0000018497C73D00; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:22:04 PM: DEBUG [usuario_dao.py:51] Buscando: rmora
05:22:04 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:22:04 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
05:22:04 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x0000018497C73D00; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:24:02 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:24:02 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000002AA790463A0>
05:24:02 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002AA78FB19D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:24:02 PM: DEBUG [servicio_dao.py:14] Seleccionando servicios...
05:24:02 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:24:02 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:24:02 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002AA78FB19D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:24:09 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:24:09 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002AA78FB19D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:24:09 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmora, None
05:24:09 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:24:09 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:24:09 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002AA78FB19D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:25:07 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:25:08 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000022B584BE460>
05:25:08 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x0000022B584309D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:25:08 PM: DEBUG [servicio_dao.py:14] Seleccionando servicios...
05:25:08 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:25:08 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:25:08 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x0000022B584309D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:25:13 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:25:13 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x0000022B584309D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:25:13 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmora, None
05:25:13 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:25:13 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:25:13 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x0000022B584309D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:26:56 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:26:56 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001C15743A460>
05:26:56 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001C1573B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:26:56 PM: DEBUG [servicio_dao.py:14] Seleccionando servicios...
05:26:56 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:26:56 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:26:56 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001C1573B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:27:01 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:27:01 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001C1573B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:27:01 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmora, None
05:27:01 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:27:01 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:27:01 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001C1573B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:27:09 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:27:09 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001C1573B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:27:09 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, jperez, None
05:27:09 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:27:09 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:27:09 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001C1573B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:30:07 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:30:08 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000001547F5FE4C0>
05:30:08 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001547F5709D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:30:08 PM: DEBUG [servicio_dao.py:14] Seleccionando servicios...
05:30:08 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:30:08 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:30:08 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001547F5709D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:30:15 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:30:15 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000001547F5709D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:30:15 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmora, None
05:30:15 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:30:15 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:30:15 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000001547F5709D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:30:54 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:30:54 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x00000285BFA43580>
05:30:54 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x00000285BF9B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:30:54 PM: DEBUG [servicio_dao.py:14] Seleccionando servicios...
05:30:54 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:30:54 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:30:54 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x00000285BF9B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:30:59 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:30:59 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x00000285BF9B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:30:59 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmora, None
05:30:59 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:30:59 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:30:59 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x00000285BF9B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:31:06 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:31:06 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x00000285BF9B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:31:06 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmora, None
05:31:06 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:31:06 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:31:06 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x00000285BF9B09D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:32:28 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:32:28 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x00000216442235B0>
05:32:28 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x00000216441909D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:32:28 PM: DEBUG [servicio_dao.py:14] Seleccionando servicios...
05:32:28 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:32:28 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:32:28 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x00000216441909D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:32:32 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:32:32 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x00000216441909D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:32:32 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, ew, None
05:32:32 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:32:32 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:32:32 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x00000216441909D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:32:37 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:32:37 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x00000216441909D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:32:37 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmora, None
05:32:37 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:32:37 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:32:37 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x00000216441909D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:33:39 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:33:39 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000002A69DEA3550>
05:33:39 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:33:39 PM: DEBUG [servicio_dao.py:14] Seleccionando servicios...
05:33:39 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:33:39 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:33:39 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:33:44 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:33:44 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:33:44 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmora, None
05:33:44 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:33:44 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:33:44 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:33:48 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:33:48 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:33:48 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmor, None
05:33:48 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:33:48 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:33:48 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:33:49 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:33:49 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:33:49 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmor, None
05:33:49 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:33:49 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:33:49 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:33:49 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:33:49 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:33:49 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmor, None
05:33:49 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:33:49 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:33:49 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:33:52 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:33:52 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:33:52 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rmor, None
05:33:52 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:33:52 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:33:52 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:33:55 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:33:55 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:33:55 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, rm, None
05:33:55 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:33:55 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:33:55 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:33:59 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:33:59 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:33:59 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, , None
05:33:59 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:33:59 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:33:59 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:33:59 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:33:59 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:33:59 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, , None
05:33:59 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:33:59 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:33:59 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:33:59 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
05:33:59 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
05:33:59 PM: DEBUG [usuario_dao.py:51] Buscando: Usuario: None, , None
05:33:59 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
05:33:59 PM: DEBUG [cursor_pool.py:22] Commit de transacción
05:33:59 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002A69DE109D0; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
05:34:21 PM: INFO [logger_base.py:13] Mensaje a nivel info
05:34:21 PM: WARNING [logger_base.py:14] Mensaje a nivel warning
05:34:21 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
05:34:21 PM: CRITICAL [logger_base.py:16] Mensaje a nivel crítico
05:45:33 PM: INFO [usuario_dao.py:70] Usuario: rmora, 13579, None
05:45:33 PM: INFO [usuario_dao.py:70] Usuario: rmora, 13579, None
05:45:59 PM: INFO [usuario_dao.py:70] Usuario: 4, rmora, 13579
05:45:59 PM: INFO [usuario_dao.py:70] Usuario: 5, rmora, 13579
07:01:47 PM: INFO [menu.py:92] 
09:20:23 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:20:55 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:21:46 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:21:56 AM: INFO [usuario_dao.py:70] Usuario: 4, rmora, 13579
09:21:56 AM: INFO [usuario_dao.py:70] Usuario: 5, rmora, 13579
09:23:53 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:24:13 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:24:23 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:24:33 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:24:48 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:25:16 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:25:27 AM: INFO [usuario_dao.py:70] Usuario: 4, rmora, 13579
09:25:27 AM: INFO [usuario_dao.py:70] Usuario: 5, rmora, 13579
09:26:13 AM: INFO [usuario_dao.py:70] Usuario: 4, rmora, 13579
09:26:13 AM: INFO [usuario_dao.py:70] Usuario: 5, rmora, 13579
09:28:16 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:28:29 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:28:40 AM: INFO [usuario_dao.py:70] Usuario: 4, rmora, 13579
09:28:40 AM: INFO [usuario_dao.py:70] Usuario: 5, rmora, 13579
09:28:47 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:29:08 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:33:10 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:34:01 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:35:28 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:35:43 AM: INFO [usuario_dao.py:70] Usuario: 4, rmora, 13579
09:35:43 AM: INFO [usuario_dao.py:70] Usuario: 5, rmora, 13579
09:35:55 AM: INFO [usuario_dao.py:70] Usuario: 4, rmora, 13579
09:35:55 AM: INFO [usuario_dao.py:70] Usuario: 5, rmora, 13579
09:37:20 AM: INFO [usuario_dao.py:70] Usuario: 9, mmendoza, 8642
04:09:01 PM: INFO [menu.py:109] 
04:16:13 PM: INFO [menu.py:109] 
04:27:04 PM: INFO [usuario_dao.py:69] Usuarios eliminados: 1
01:39:55 PM: INFO [menu.py:115] 
01:39:59 PM: INFO [menu.py:115] 
01:42:09 PM: INFO [menu.py:115] 
06:29:21 PM: INFO [login.py:43] []
06:31:20 PM: INFO [login.py:43] [<usuario.Usuario object at 0x000002237EFA5DF0>]
06:41:19 PM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
09:40:08 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
10:10:16 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
10:10:32 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
10:12:00 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
10:12:03 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
10:12:04 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
10:21:04 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
10:09:42 AM: ERROR [cursor_pool.py:19] Ocurrió un error, se hizo rollback
07:35:23 PM: DEBUG [logger_base.py:12] Mensaje a nivel debug
07:35:23 PM: INFO [logger_base.py:13] Mensaje a nivel info
07:35:23 PM: WARNING [logger_base.py:14] Mensaje a nivel warning
07:35:23 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
07:35:23 PM: CRITICAL [logger_base.py:16] Mensaje a nivel crítico
07:35:32 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x000002045B4F5D60>
07:35:32 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x000002045B54FE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
07:35:32 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x000002045B54FE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
07:36:23 PM: INFO [logger_base.py:13] Mensaje a nivel info
07:36:23 PM: WARNING [logger_base.py:14] Mensaje a nivel warning
07:36:23 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
07:36:23 PM: CRITICAL [logger_base.py:16] Mensaje a nivel crítico
07:36:51 PM: INFO [logger_base.py:13] Mensaje a nivel info
07:36:51 PM: WARNING [logger_base.py:14] Mensaje a nivel warning
07:36:51 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
07:36:51 PM: CRITICAL [logger_base.py:16] Mensaje a nivel crítico
08:44:06 PM: INFO [logger_base.py:13] Mensaje a nivel info
08:44:06 PM: WARNING [logger_base.py:14] Mensaje a nivel warning
08:44:06 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
08:44:06 PM: CRITICAL [logger_base.py:16] Mensaje a nivel crítico
08:44:34 PM: DEBUG [logger_base.py:12] Mensaje a nivel debug
08:44:34 PM: INFO [logger_base.py:13] Mensaje a nivel info
08:44:34 PM: WARNING [logger_base.py:14] Mensaje a nivel warning
08:44:34 PM: ERROR [logger_base.py:15] Mensaje a nivel de error
08:44:34 PM: CRITICAL [logger_base.py:16] Mensaje a nivel crítico
08:44:40 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000027E11695D60>
08:44:40 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x0000027E116EFE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
08:44:40 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x0000027E116EFE10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
08:44:52 PM: DEBUG [cursor_pool.py:10] Inicio el método del with __enter__
08:44:52 PM: DEBUG [conexion.py:25] Creación de poool exitosa: <psycopg2.pool.SimpleConnectionPool object at 0x0000025F02CE4DC0>
08:44:52 PM: DEBUG [conexion.py:36] Conexión obtenida del Pool: <connection object at 0x0000025F02D24E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0>
08:44:52 PM: DEBUG [cursor_pool.py:28] Dentro del bloque With
08:44:52 PM: DEBUG [cursor_pool.py:30] [(1, 'jperez', '123'), (2, 'kgomez', '456'), (4, 'rmora', '13579'), (8, 'psoto', '9753'), (9, 'mmendoza', '8642'), (10, 'jrodriguez', ''), (12, 'fcajas', '1928'), (13, 'frecillas', '3456'), (14, 'mcordero', '3142'), (15, 'lquiroz', '9768'), (18, 'ymorales', '1946'), (21, 'Root', 'root')]
08:44:52 PM: DEBUG [cursor_pool.py:16] Se ejecuta el método __exit__
08:44:52 PM: DEBUG [cursor_pool.py:22] Commit de transacción
08:44:52 PM: DEBUG [conexion.py:42] Conexión <connection object at 0x0000025F02D24E10; dsn: 'user=postgres password=xxx dbname=test_db host=127.0.0.1 port=5432', closed: 0> liberada
